bokehjs/src/lib/models/glyphs/defs.ts

```diff
@@ -60,6 +61,26 @@ function _one_hex(ctx: Context2d, r: number): void {
   ctx.closePath()
 }
 
+function _one_star(ctx: Context2d, r: number): void {
+  const a = Math.sqrt(5-2*SQ5)*r
+  const c36 = (SQ5+1)/4
+  const s36 = Math.sqrt((5-SQ5)/8)
+  const c72 = (SQ5-1)/4
+  const s72 = Math.sqrt((5+SQ5)/8)
```

Comment:
The values that don't depend on `r` can be moved to the top of the module (with all-caps naming)